src/app/static/src/app/localStorageManager.ts

```diff
@@ -35,6 +35,8 @@ export class LocalStorageManager {
     };
 
     getState = (): Partial<RootState> | null => {
+        // console.log('storage manager currentUser', state.currentUser)
+        // console.log('storage manager ', this)
```

Comment:
I'm not sure if it is possible to access the root store from here so I have left this file as is (ie, declaring currentUser). Let me know if you think there is another way of doing this.

Reply:
That seems correct to me :+1: Just need to delete the 2 lines of commented code.